docs/src/fileformat.md

```diff
@@ -133,7 +133,7 @@ The following convenience "macros"/source placeholders are always expanded:
 
 - `@__NAME__`:
 
-  expands to the `name` keyword argument to [`Literate.markdown`](@ref),
```

Comment:
I don't agree with this one, the full sentence is
```
@__NAME__ expands to the name keyword argument
```
isn't that correct?

Reply:
ah, yes that is correct - we should ignore my "fix" here

Reply:
Happy to help, the docs are awesome!

I'm planning to use Literate.jl for my Franklin.jl site, and the docs made it so easy.

Reply:
Good. I spent quite some time writing it, but you never really get much feedback so difficult to know if they are good or bad. Thanks again.